src/views/VallisMap.vue

```diff
@@ -159,175 +121,192 @@ function onEachOddity(feature, layer) {
           },
+          onEachFeature: onEachFeature,
         },
-      ],
-      caves: {
-        fishcave: {
-          loc: L.latLng(1895.73, 1480.74),
-          title: 'Fishing Cave',
-          position: 'bottom',
-        },
-        toroidfishcave: {
-          json: toroidFishCave,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: fishToroidCaveMarker });
-            },
-            onEachFeature: onEachFeature,
+      },
+      {
+        name: 'Oddity',
+        json: oddity,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: oddityMarker });
           },
+          onEachFeature: onEachOddity,
         },
-        toroidcave: {
-          json: toroidCave,
-          opts: {
-            pointToLayer: function(feature, latlng) {
-              return markerAlias(latlng, { icon: toroidCaveMarker });
-            },
-            onEachFeature: onEachFeature,
+      },
+      {
+        name: 'Somachord Tone',
+        json: somachord,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: somachordMarker });
           },
+          onEachFeature: onEachFeature,
         },
       },
-      icons: {
-        calda: {
-          src: caldaIcon,
-          size: [90, 62],
-        },
-        sola: {
-          src: solaIcon,
-          size: [90, 62],
-        },
-        vega: {
-          src: vegaIcon,
-          size: [90, 62],
-        },
-        fishcave: {
-          src: fishCaveIcon,
-          size: [50, 34],
+      {
+        name: 'Toroids',
+        json: toroids,
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: toroidMarkerFromName(feature.properties.name) });
+          },
+          onEachFeature: onEachFeature,
         },
       },
-      markers: {
-        calda: {
-          loc: L.latLng(642.88, 688.49),
-          title: 'Calda Toroid',
-          position: 'bottom',
-        },
-        sola: {
-          loc: L.latLng(1871.63, 550.59),
-          title: 'Sola Toroid',
-          position: 'bottom',
-        },
-        vega: {
-          loc: L.latLng(440.62, 1455.98),
-          title: 'Vega Toroid',
-          position: 'bottom',
+      {
+        name: 'Special Caves',
+        json: fishCave.concat(toroidCave).concat(toroidFishCave),
+        opts: {
+          pointToLayer: function(feature, latlng) {
+            return markerAlias(latlng, { icon: caveMarkerFromName(feature.properties.name) });
+          },
+          onEachFeature: onEachFeature,
         },
       },
-    };
-  },
+    ],
+  };
+}
+
+export default {
+  name: 'Vallismap',
+  data: dataFn,
   methods: {
     track() {
       this.$ga.page('/vallis/map');
     },
   },
-  mounted: function() {},
+  mounted: function() {
```

Comment:
if we're going to abstract the data function out to a separate one (presumably for space/organization?) could you move mounted to one as well?

Reply:
Yep!